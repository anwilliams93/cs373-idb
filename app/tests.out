test_challenge_run_0 (api.api_test.TestAPI) ... ok
test_challenge_run_1 (api.api_test.TestAPI) ... ok
test_challenge_run_4 (api.api_test.TestAPI) ... ok
test_challenge_themes_0 (api.api_test.TestAPI) ... ok
test_challenge_themes_1 (api.api_test.TestAPI) ... ok
test_challenge_themes_2 (api.api_test.TestAPI) ... ok
test_challenges_by_id_0 (api.api_test.TestAPI) ... ok
test_challenges_by_id_1 (api.api_test.TestAPI) ... ok
test_challenges_by_id_2 (api.api_test.TestAPI) ... ok
test_get_challenges (api.api_test.TestAPI) ... ok
test_get_challenges_slash (api.api_test.TestAPI) ... ok
test_get_run_by_id_0 (api.api_test.TestAPI) ... ok
test_get_run_by_id_0_slash (api.api_test.TestAPI) ... ok
test_get_run_by_id_1 (api.api_test.TestAPI) ... ok
test_get_run_by_id_2 (api.api_test.TestAPI) ... ok
test_get_run_challenges_0 (api.api_test.TestAPI) ... ok
test_get_run_challenges_1 (api.api_test.TestAPI) ... ok
test_get_run_challenges_2 (api.api_test.TestAPI) ... ok
test_get_run_themes_0 (api.api_test.TestAPI) ... ok
test_get_run_themes_1 (api.api_test.TestAPI) ... ok
test_get_run_themes_2 (api.api_test.TestAPI) ... ok
test_get_runs (api.api_test.TestAPI) ... ok
test_get_runs_slash (api.api_test.TestAPI) ... ok
test_get_themes (api.api_test.TestAPI) ... ok
test_get_themes_slash (api.api_test.TestAPI) ... ok
test_locations (api.api_test.TestAPI) ... ok
test_locations_by_id_0 (api.api_test.TestAPI) ... ok
test_locations_by_id_1 (api.api_test.TestAPI) ... ok
test_locations_by_id_2 (api.api_test.TestAPI) ... ok
test_locations_by_id_3 (api.api_test.TestAPI) ... ok
test_root (api.api_test.TestAPI) ... ok
test_root_slash (api.api_test.TestAPI) ... ok
test_theme_challenges_0 (api.api_test.TestAPI) ... ok
test_theme_challenges_0_slash (api.api_test.TestAPI) ... ok
test_theme_challenges_1 (api.api_test.TestAPI) ... ok
test_theme_challenges_2 (api.api_test.TestAPI) ... ok
test_theme_run_0 (api.api_test.TestAPI) ... ok
test_theme_run_1 (api.api_test.TestAPI) ... ok
test_theme_run_2 (api.api_test.TestAPI) ... ok
test_themes_by_id_0 (api.api_test.TestAPI) ... ok
test_themes_by_id_0_slash (api.api_test.TestAPI) ... ok
test_themes_by_id_1 (api.api_test.TestAPI) ... ok
test_themes_by_id_1_slash (api.api_test.TestAPI) ... ok
test_themes_by_id_2 (api.api_test.TestAPI) ... ok

----------------------------------------------------------------------
Ran 44 tests in 0.573s

OK
test_sort_difficulty_asc (api.api_sorting_challenges_test.TestChallengesSort) ... ok
test_sort_difficulty_desc (api.api_sorting_challenges_test.TestChallengesSort) ... ok
test_sort_name_asc (api.api_sorting_challenges_test.TestChallengesSort) ... ok
test_sort_name_desc (api.api_sorting_challenges_test.TestChallengesSort) ... ok
test_sort_numberOfRuns_asc (api.api_sorting_challenges_test.TestChallengesSort) ... ok
test_sort_numberOfRuns_desc (api.api_sorting_challenges_test.TestChallengesSort) ... ok
test_sort_numberOfThemes_asc (api.api_sorting_challenges_test.TestChallengesSort) ... ok
test_sort_numberOfThemes_desc (api.api_sorting_challenges_test.TestChallengesSort) ... ok

----------------------------------------------------------------------
Ran 8 tests in 0.580s

OK
test_sort_altitude_asc (api.api_sorting_locations_test.TestLocationsSort) ... ok
test_sort_altitude_desc (api.api_sorting_locations_test.TestLocationsSort) ... ok
test_sort_fallAvgHum_asc (api.api_sorting_locations_test.TestLocationsSort) ... ok
test_sort_fallAvgHum_desc (api.api_sorting_locations_test.TestLocationsSort) ... ok
test_sort_fallAvgTemp_asc (api.api_sorting_locations_test.TestLocationsSort) ... ok
test_sort_fallAvgTemp_desc (api.api_sorting_locations_test.TestLocationsSort) ... ok
test_sort_names_asc (api.api_sorting_locations_test.TestLocationsSort) ... ok
test_sort_names_desc (api.api_sorting_locations_test.TestLocationsSort) ... ok
test_sort_springAvgHum_asc (api.api_sorting_locations_test.TestLocationsSort) ... ok
test_sort_springAvgHum_desc (api.api_sorting_locations_test.TestLocationsSort) ... ok
test_sort_springAvgTemp_asc (api.api_sorting_locations_test.TestLocationsSort) ... ok
test_sort_springAvgTemp_desc (api.api_sorting_locations_test.TestLocationsSort) ... ok
test_sort_summerAvgHum_asc (api.api_sorting_locations_test.TestLocationsSort) ... ok
test_sort_summerAvgHum_desc (api.api_sorting_locations_test.TestLocationsSort) ... ok
test_sort_summerAvgTemp_asc (api.api_sorting_locations_test.TestLocationsSort) ... ok
test_sort_summerAvgTemp_desc (api.api_sorting_locations_test.TestLocationsSort) ... ok
test_sort_winterAvgHum_asc (api.api_sorting_locations_test.TestLocationsSort) ... ok
test_sort_winterAvgHum_desc (api.api_sorting_locations_test.TestLocationsSort) ... ok
test_sort_winterAvgTemp_asc (api.api_sorting_locations_test.TestLocationsSort) ... ok
test_sort_winterAvgTemp_desc (api.api_sorting_locations_test.TestLocationsSort) ... ok

----------------------------------------------------------------------
Ran 20 tests in 0.515s

OK
test_filter_buzzword (api.api_filtering_themes_test.TestThemesFilter) ... ok
test_filter_challenge (api.api_filtering_themes_test.TestThemesFilter) ... ok
test_filter_funrun (api.api_filtering_themes_test.TestThemesFilter) ... ok
test_filter_two_buzzwords (api.api_filtering_themes_test.TestThemesFilter) ... ok
test_filter_two_challenges (api.api_filtering_themes_test.TestThemesFilter) ... ok
test_filter_two_funruns (api.api_filtering_themes_test.TestThemesFilter) ... ok

----------------------------------------------------------------------
Ran 6 tests in 0.150s

OK
test_challenge_filter_run1 (api.api_filtering_challenges_test.TestChallengeFilter) ... ok
test_challenge_filter_run10 (api.api_filtering_challenges_test.TestChallengeFilter) ... ok
test_challenge_filter_run11 (api.api_filtering_challenges_test.TestChallengeFilter) ... ok
test_challenge_filter_run12 (api.api_filtering_challenges_test.TestChallengeFilter) ... ok
test_challenge_filter_run13 (api.api_filtering_challenges_test.TestChallengeFilter) ... ok
test_challenge_filter_run14 (api.api_filtering_challenges_test.TestChallengeFilter) ... ok
test_challenge_filter_run15 (api.api_filtering_challenges_test.TestChallengeFilter) ... ok
test_challenge_filter_run16 (api.api_filtering_challenges_test.TestChallengeFilter) ... ok
test_challenge_filter_run17 (api.api_filtering_challenges_test.TestChallengeFilter) ... ok
test_challenge_filter_run18 (api.api_filtering_challenges_test.TestChallengeFilter) ... ok
test_challenge_filter_run2 (api.api_filtering_challenges_test.TestChallengeFilter) ... ok
test_challenge_filter_run3 (api.api_filtering_challenges_test.TestChallengeFilter) ... ok
test_challenge_filter_run4 (api.api_filtering_challenges_test.TestChallengeFilter) ... ok
test_challenge_filter_run5 (api.api_filtering_challenges_test.TestChallengeFilter) ... ok
test_challenge_filter_run6 (api.api_filtering_challenges_test.TestChallengeFilter) ... ok
test_challenge_filter_run7 (api.api_filtering_challenges_test.TestChallengeFilter) ... ok
test_challenge_filter_run8 (api.api_filtering_challenges_test.TestChallengeFilter) ... ok
test_challenge_filter_run9 (api.api_filtering_challenges_test.TestChallengeFilter) ... ok

----------------------------------------------------------------------
Ran 18 tests in 0.666s

OK
test_filter_1 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_10 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_11 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_12 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_13 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_14 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_15 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_16 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_17 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_18 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_19 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_2 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_20 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_21 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_22 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_23 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_24 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_25 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_26 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_27 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_28 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_29 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_3 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_30 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_4 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_5 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_6 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_7 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_8 (api.loc_filter_test.TestLocationFilter) ... ok
test_filter_9 (api.loc_filter_test.TestLocationFilter) ... ok

----------------------------------------------------------------------
Ran 30 tests in 0.420s

OK

test_add_challenge (models.models_test.TestDBModels) ... ok
test_add_funrun (models.models_test.TestDBModels) ... ok
test_add_location (models.models_test.TestDBModels) ... ok
test_add_themes (models.models_test.TestDBModels) ... ok
test_challenges (models.models_test.TestDBModels) ... ok
test_fun_challenge_relationship_1 (models.models_test.TestDBModels) ... ok
test_fun_challenge_relationship_2 (models.models_test.TestDBModels) ... ok
test_fun_challenge_relationship_3 (models.models_test.TestDBModels) ... ok
test_fun_location_relationship_1 (models.models_test.TestDBModels) ... ok
test_fun_location_relationship_2 (models.models_test.TestDBModels) ... ok
test_fun_location_relationship_3 (models.models_test.TestDBModels) ... ok
test_fun_location_relationship_4 (models.models_test.TestDBModels) ... ok
test_fun_theme_relationship_1 (models.models_test.TestDBModels) ... ok
test_fun_theme_relationship_2 (models.models_test.TestDBModels) ... ok
test_fun_theme_relationship_3 (models.models_test.TestDBModels) ... ok
test_fun_theme_relationship_4 (models.models_test.TestDBModels) ... ok
test_funrun (models.models_test.TestDBModels) ... ok
test_location (models.models_test.TestDBModels) ... ok
test_theme_challenge_relationship_1 (models.models_test.TestDBModels) ... ok
test_theme_challenge_relationship_2 (models.models_test.TestDBModels) ... ok
test_theme_challenge_relationship_3 (models.models_test.TestDBModels) ... ok
test_theme_challenge_relationship_4 (models.models_test.TestDBModels) ... ok
test_themes (models.models_test.TestDBModels) ... ok

----------------------------------------------------------------------
Ran 23 tests in 0.132s

OK