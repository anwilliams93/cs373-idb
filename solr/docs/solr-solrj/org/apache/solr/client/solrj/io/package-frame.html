<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_67) on Thu Apr 09 10:38:05 MDT 2015 -->
<meta http-equiv="Content-Type" content="text/html" charset="utf-8">
<title>org.apache.solr.client.solrj.io (Solr 5.1.0 API)</title>
<meta name="date" content="2015-04-09">
<link rel="stylesheet" type="text/css" href="../../../../../../stylesheet.css" title="Style">
</head>
<body>
<h1 class="bar"><a href="../../../../../../org/apache/solr/client/solrj/io/package-summary.html" target="classFrame">org.apache.solr.client.solrj.io</a></h1>
<div class="indexContainer">
<h2 title="Classes">Classes</h2>
<ul title="Classes">
<li><a href="AscFieldComp.html" title="class in org.apache.solr.client.solrj.io" target="classFrame">AscFieldComp</a></li>
<li><a href="CloudSolrStream.html" title="class in org.apache.solr.client.solrj.io" target="classFrame">CloudSolrStream</a></li>
<li><a href="DescFieldComp.html" title="class in org.apache.solr.client.solrj.io" target="classFrame">DescFieldComp</a></li>
<li><a href="JSONTupleStream.html" title="class in org.apache.solr.client.solrj.io" target="classFrame">JSONTupleStream</a></li>
<li><a href="MergeStream.html" title="class in org.apache.solr.client.solrj.io" target="classFrame">MergeStream</a></li>
<li><a href="MultiComp.html" title="class in org.apache.solr.client.solrj.io" target="classFrame">MultiComp</a></li>
<li><a href="ParallelStream.html" title="class in org.apache.solr.client.solrj.io" target="classFrame">ParallelStream</a></li>
<li><a href="PushBackStream.html" title="class in org.apache.solr.client.solrj.io" target="classFrame">PushBackStream</a></li>
<li><a href="RankStream.html" title="class in org.apache.solr.client.solrj.io" target="classFrame">RankStream</a></li>
<li><a href="ReducerStream.html" title="class in org.apache.solr.client.solrj.io" target="classFrame">ReducerStream</a></li>
<li><a href="SolrClientCache.html" title="class in org.apache.solr.client.solrj.io" target="classFrame">SolrClientCache</a></li>
<li><a href="SolrStream.html" title="class in org.apache.solr.client.solrj.io" target="classFrame">SolrStream</a></li>
<li><a href="StreamContext.html" title="class in org.apache.solr.client.solrj.io" target="classFrame">StreamContext</a></li>
<li><a href="Tuple.html" title="class in org.apache.solr.client.solrj.io" target="classFrame">Tuple</a></li>
<li><a href="TupleStream.html" title="class in org.apache.solr.client.solrj.io" target="classFrame">TupleStream</a></li>
<li><a href="UniqueStream.html" title="class in org.apache.solr.client.solrj.io" target="classFrame">UniqueStream</a></li>
</ul>
</div>
</body>
</html>
