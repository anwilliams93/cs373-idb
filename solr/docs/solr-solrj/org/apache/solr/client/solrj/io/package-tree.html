<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_67) on Thu Apr 09 10:38:05 MDT 2015 -->
<meta http-equiv="Content-Type" content="text/html" charset="utf-8">
<title>org.apache.solr.client.solrj.io Class Hierarchy (Solr 5.1.0 API)</title>
<meta name="date" content="2015-04-09">
<link rel="stylesheet" type="text/css" href="../../../../../../stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="org.apache.solr.client.solrj.io Class Hierarchy (Solr 5.1.0 API)";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li>Class</li>
<li>Use</li>
<li class="navBarCell1Rev">Tree</li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="../../../../../../org/apache/solr/client/solrj/impl/package-tree.html">Prev</a></li>
<li><a href="../../../../../../org/apache/solr/client/solrj/request/package-tree.html">Next</a></li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?org/apache/solr/client/solrj/io/package-tree.html" target="_top">Frames</a></li>
<li><a href="package-tree.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<div class="header">
<h1 class="title">Hierarchy For Package org.apache.solr.client.solrj.io</h1>
<span class="strong">Package Hierarchies:</span>
<ul class="horizontal">
<li><a href="../../../../../../overview-tree.html">All Packages</a></li>
</ul>
</div>
<div class="contentContainer">
<h2 title="Class Hierarchy">Class Hierarchy</h2>
<ul>
<li type="circle">java.lang.<a href="http://download.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang"><span class="strong">Object</span></a>
<ul>
<li type="circle">org.apache.solr.client.solrj.io.<a href="../../../../../../org/apache/solr/client/solrj/io/AscFieldComp.html" title="class in org.apache.solr.client.solrj.io"><span class="strong">AscFieldComp</span></a> (implements java.util.<a href="http://download.oracle.com/javase/7/docs/api/java/util/Comparator.html?is-external=true" title="class or interface in java.util">Comparator</a>&lt;T&gt;, java.io.<a href="http://download.oracle.com/javase/7/docs/api/java/io/Serializable.html?is-external=true" title="class or interface in java.io">Serializable</a>)</li>
<li type="circle">org.apache.solr.client.solrj.io.<a href="../../../../../../org/apache/solr/client/solrj/io/CloudSolrStream.StreamOpener.html" title="class in org.apache.solr.client.solrj.io"><span class="strong">CloudSolrStream.StreamOpener</span></a> (implements java.util.concurrent.<a href="http://download.oracle.com/javase/7/docs/api/java/util/concurrent/Callable.html?is-external=true" title="class or interface in java.util.concurrent">Callable</a>&lt;V&gt;)</li>
<li type="circle">org.apache.solr.client.solrj.io.<a href="../../../../../../org/apache/solr/client/solrj/io/CloudSolrStream.TupleWrapper.html" title="class in org.apache.solr.client.solrj.io"><span class="strong">CloudSolrStream.TupleWrapper</span></a> (implements java.lang.<a href="http://download.oracle.com/javase/7/docs/api/java/lang/Comparable.html?is-external=true" title="class or interface in java.lang">Comparable</a>&lt;T&gt;)</li>
<li type="circle">org.apache.solr.client.solrj.io.<a href="../../../../../../org/apache/solr/client/solrj/io/DescFieldComp.html" title="class in org.apache.solr.client.solrj.io"><span class="strong">DescFieldComp</span></a> (implements java.util.<a href="http://download.oracle.com/javase/7/docs/api/java/util/Comparator.html?is-external=true" title="class or interface in java.util">Comparator</a>&lt;T&gt;, java.io.<a href="http://download.oracle.com/javase/7/docs/api/java/io/Serializable.html?is-external=true" title="class or interface in java.io">Serializable</a>)</li>
<li type="circle">org.apache.solr.client.solrj.io.<a href="../../../../../../org/apache/solr/client/solrj/io/JSONTupleStream.html" title="class in org.apache.solr.client.solrj.io"><span class="strong">JSONTupleStream</span></a></li>
<li type="circle">org.apache.solr.client.solrj.io.<a href="../../../../../../org/apache/solr/client/solrj/io/MultiComp.html" title="class in org.apache.solr.client.solrj.io"><span class="strong">MultiComp</span></a> (implements java.util.<a href="http://download.oracle.com/javase/7/docs/api/java/util/Comparator.html?is-external=true" title="class or interface in java.util">Comparator</a>&lt;T&gt;, java.io.<a href="http://download.oracle.com/javase/7/docs/api/java/io/Serializable.html?is-external=true" title="class or interface in java.io">Serializable</a>)</li>
<li type="circle">org.apache.solr.client.solrj.io.<a href="../../../../../../org/apache/solr/client/solrj/io/SolrClientCache.html" title="class in org.apache.solr.client.solrj.io"><span class="strong">SolrClientCache</span></a> (implements java.io.<a href="http://download.oracle.com/javase/7/docs/api/java/io/Serializable.html?is-external=true" title="class or interface in java.io">Serializable</a>)</li>
<li type="circle">org.apache.solr.client.solrj.io.<a href="../../../../../../org/apache/solr/client/solrj/io/StreamContext.html" title="class in org.apache.solr.client.solrj.io"><span class="strong">StreamContext</span></a> (implements java.io.<a href="http://download.oracle.com/javase/7/docs/api/java/io/Serializable.html?is-external=true" title="class or interface in java.io">Serializable</a>)</li>
<li type="circle">org.apache.solr.client.solrj.io.<a href="../../../../../../org/apache/solr/client/solrj/io/Tuple.html" title="class in org.apache.solr.client.solrj.io"><span class="strong">Tuple</span></a> (implements java.lang.<a href="http://download.oracle.com/javase/7/docs/api/java/lang/Cloneable.html?is-external=true" title="class or interface in java.lang">Cloneable</a>)</li>
<li type="circle">org.apache.solr.client.solrj.io.<a href="../../../../../../org/apache/solr/client/solrj/io/TupleStream.html" title="class in org.apache.solr.client.solrj.io"><span class="strong">TupleStream</span></a> (implements java.io.<a href="http://download.oracle.com/javase/7/docs/api/java/io/Serializable.html?is-external=true" title="class or interface in java.io">Serializable</a>)
<ul>
<li type="circle">org.apache.solr.client.solrj.io.<a href="../../../../../../org/apache/solr/client/solrj/io/CloudSolrStream.html" title="class in org.apache.solr.client.solrj.io"><span class="strong">CloudSolrStream</span></a>
<ul>
<li type="circle">org.apache.solr.client.solrj.io.<a href="../../../../../../org/apache/solr/client/solrj/io/ParallelStream.html" title="class in org.apache.solr.client.solrj.io"><span class="strong">ParallelStream</span></a></li>
</ul>
</li>
<li type="circle">org.apache.solr.client.solrj.io.<a href="../../../../../../org/apache/solr/client/solrj/io/MergeStream.html" title="class in org.apache.solr.client.solrj.io"><span class="strong">MergeStream</span></a></li>
<li type="circle">org.apache.solr.client.solrj.io.<a href="../../../../../../org/apache/solr/client/solrj/io/PushBackStream.html" title="class in org.apache.solr.client.solrj.io"><span class="strong">PushBackStream</span></a></li>
<li type="circle">org.apache.solr.client.solrj.io.<a href="../../../../../../org/apache/solr/client/solrj/io/RankStream.html" title="class in org.apache.solr.client.solrj.io"><span class="strong">RankStream</span></a></li>
<li type="circle">org.apache.solr.client.solrj.io.<a href="../../../../../../org/apache/solr/client/solrj/io/ReducerStream.html" title="class in org.apache.solr.client.solrj.io"><span class="strong">ReducerStream</span></a></li>
<li type="circle">org.apache.solr.client.solrj.io.<a href="../../../../../../org/apache/solr/client/solrj/io/SolrStream.html" title="class in org.apache.solr.client.solrj.io"><span class="strong">SolrStream</span></a></li>
<li type="circle">org.apache.solr.client.solrj.io.<a href="../../../../../../org/apache/solr/client/solrj/io/UniqueStream.html" title="class in org.apache.solr.client.solrj.io"><span class="strong">UniqueStream</span></a></li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li>Class</li>
<li>Use</li>
<li class="navBarCell1Rev">Tree</li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="../../../../../../org/apache/solr/client/solrj/impl/package-tree.html">Prev</a></li>
<li><a href="../../../../../../org/apache/solr/client/solrj/request/package-tree.html">Next</a></li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?org/apache/solr/client/solrj/io/package-tree.html" target="_top">Frames</a></li>
<li><a href="package-tree.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
<p class="legalCopy"><small>
          <i>Copyright &copy; 2000-2015 Apache Software Foundation.  All Rights Reserved.</i>
          <script src='../../../../../../prettify.js' type='text/javascript'></script>
          <script type='text/javascript'>
            (function(){
              var oldonload = window.onload;
              if (typeof oldonload != 'function') {
                window.onload = prettyPrint;
              } else {
                window.onload = function() {
                  oldonload();
                  prettyPrint();
                }
              }
            })();
          </script>
        </small></p>
</body>
</html>
