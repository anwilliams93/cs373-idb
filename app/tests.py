from io       import StringIO
from unittest import main, TestCase
import urllib.request, codecs, json

server_address = 'http://104.239.139.43:8000'

def http_response_to_json_object (response) :
	reader = codecs.getreader("utf-8")
	return json.load(reader(response))

class TestAPI (TestCase) :

	def test_root_1 (self) :
		url = server_address + '/api'
		response = urllib.request.urlopen(url)
		response_object = http_response_to_json_object(response)
		expected = 	{'urls':	{
			              		"funruns_url": "/funruns",
			                	"themes_url": "/themes",
			                	"challenges_url": "/challenges"
			            		}
					}
		self.assertEqual(list(response_object), list(expected))

if __name__ == "__main__" :
    main()