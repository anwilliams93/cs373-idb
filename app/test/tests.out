............................................
----------------------------------------------------------------------
Ran 44 tests in 1.232s

OK
Name             Stmts   Miss  Cover   Missing
----------------------------------------------
app/test/tests     276    276     0%   1-5313
